### PR TITLE
fix: update the notebook with the new Eval classes and clean some typos

### LINKED ISSUE #652: [Bug]: Gemini evaluation: string format error and missing class definition
### File Name

https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/evaluation/get_started_with_genai_model_eval_service.ipynb

### What happened?

1. The string prompt_end gives an error in "Evaluate prompt engineering using locally-defined custom metrics" section. The fix: Add f at the beginning of first prompt_end and delete the fourth quotation mark at the end of the second prompt_end.
2. Cosmetics: Delete <your-bucket-name> from BUCKET_URI definition.
3. Metric list gives an error in "Validate prompt by comparing LLM 1 with LLM 2". The fix: Delete rouge_1 and rouge_2 from metrics.
4. class EvaluationTextSummarizationSpec in "Using Vertex AI Model Evaluation Computation-based metrics" is not imported from language_models and gives an error. It doesn't exist in this version of the module (it is in preview).
Thanks! 

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by inardini**
# Description

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [x] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [x] You are listed as the author in your notebook or README file.
  - [x] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [x] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [x] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [x] Appropriate docs were updated (if necessary)

Fixes #652 🦕

### CODE REVIEW

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb [hunk 1]
```diff
@@ -308,9 +308,9 @@
    },
    "outputs": [],
    "source": [
-    "from google.colab import auth\n",
+    "# from google.colab import auth\n",
     "\n",
-    "auth.authenticate_user()"
+    "# auth.authenticate_user()"
```
Comment (holtskinner) [inline gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:313]: Can you uncomment these or change it to check if the environment is colab? This way, users can just choose not to run the cell if not using colab.

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb [hunk 2]
```diff
@@ -824,7 +835,32 @@
    },
    "outputs": [],
    "source": [
-    "llm1_model = GenerativeModel(\"gemini-pro\")\n",
+    "generation_config = {\n",
```
Comment (holtskinner) [inline gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:838]: Can you use the `GenerationConfig` class?

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb [hunk 3]
```diff
@@ -824,7 +835,32 @@
...
+    "safety_settings = {\n",
+    "    HarmCategory.HARM_CATEGORY_UNSPECIFIED: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_HATE_SPEECH: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_HARASSMENT: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT: HarmBlockThreshold.BLOCK_NONE,\n",
+    "}\n",
+    "\n",
+    "llm1_model = GenerativeModel(\n",
+    "    \"gemini-pro\", generation_config=generation_config, safety_settings=safety_settings\n",
```
Comment (holtskinner) [inline gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:852]: Specify which gemini pro model is being used. 1.0 or 1.5

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb [hunk 4]
```diff
@@ -1125,34 +1124,41 @@
     "            prompt_end = \"\"\"Examine the text and determine whether the text is faithful or not.\n",
     "                          Faithfulness refers to how accurately a generated summary reflects the essential information and key concepts present in the original source document.\n",
     "                          A faithful summary stays true to the facts and meaning of the source text, without introducing distortions, hallucinations, or information that wasn't originally there.\n",
-    "                          Your response must be single integer number on a scale of 0-5, 0 the least faithful and 5 being the most faithful.\"\"\" \n",
-    "        \n",
+    "                          Your response must be single integer number on a scale of 0-5, 0 the least faithful and 5 being the most faithful.\"\"\"\n",
```
Comment (holtskinner) [inline gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:1127]:
```suggestion
    "                          Your response must be a single integer number on a scale of 0-5, 0 the least faithful and 5 being the most faithful.\"\"\"\n",
```

**Review by holtskinner — APPROVED** [review]

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb (hunk 1 above)
Comment (inardini) [reply gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:313]: done

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb (hunk 3 above)
Comment (inardini) [reply gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:852]: done

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb (hunk 4 above)
Comment (inardini) [reply gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:1127]: done

File: gemini/evaluation/get_started_with_genai_model_eval_service.ipynb (hunk 2 above)
Comment (inardini) [reply gemini/evaluation/get_started_with_genai_model_eval_service.ipynb:838]: done

**Pull request merged**